vdb-bench-assembly/app/vdb-bench-app.js

```diff
@@ -174,6 +174,35 @@ var App;
             .when('/login', {
                 templateUrl: App.templatePath + 'login.html'
             });
+        
+        // configure i18n
+        $translateProvider.useSanitizeValueStrategy('sanitize');
+        $translateProvider.useStaticFilesLoader({
+            prefix: 'app/i18n/messages-',
+            suffix: '.json'
+        });
+        
+        // default all en_* and every other locale to en right now
+        $translateProvider.registerAvailableLanguageKeys(['en'], {
+            'en_*': 'en',
+            '*': 'en'
+         });
+        
+        // try to get locale from browser
+        var l_lang;
+        if (navigator.languages != undefined) { // Chrome
+        	l_lang = navigator.languages[0];
+        } else if (navigator.userLanguage) { // Explorer
+            l_lang = navigator.userLanguage;
+        } else if (navigator.language) { // FF
+            l_lang = navigator.language;
+        } else {
+        	l_lang = 'en'; // fallback
+        }
+        
+        // must set lang
+        $translateProvider.preferredLanguage(l_lang);
+//        $translate.fallbackLanguage('en') // seems to break things
```

Comment:
Any thoughts on why this breaks things?
Stupid question: is it because its using '$translate' rather than '$translateProvider'?
Otherwise, may want to just remove the comment for the time-being. If we need a fallback then it can be re-explored and if necessary post an issue to the parent-translate project if things are still failing.